.github/workflows/check-terraform-module-version.yaml

```diff
@@ -8,8 +8,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check for editted files for terraform modules
-        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@add-gh-action-to-pin-terraform-modules-to-latest 
+      - name: Check editted files for terraform modules
+        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@main
```

Comment:
Do we want to pin the version?

Reply:
Given that it is only our team using the action and currently we are only using it within this repository at the minute, I think it's ok to have it point at 'main'